src/controllers/user.controller.js

```diff
@@ -57,12 +60,16 @@ export default class UserController {
           console.log(err);
           return res.status(500).json({ message: 'Something went wrong!' });
         });
-      return res.status(201).json({
-        message:
-          'User registered successfully! Please check your email for verification.',
-        token
-      });
+      return res
+        .status(201)
+        .json({
+          message:
+            'User registered successfully! Please check your email for verification.',
+          token,
+          data: getUser
```

Comment:
Returning user data on registration is not necessary. Status and token and message are enough.